packages/ui-library/src/components/ui/divider/index.stories.ts

```diff
@@ -25,18 +49,54 @@ export default {
       canvas: { hidden: true },
     },
     viewMode: 'docs',
+    layout: 'centered',
+    docs: {
+      description: {
+        component: `
+  <Markdown>
```

Comment:
Changing it removed one warning but now it says:"The tag <markdown> is unrecognized in this browser. " which isn't really true since the tag has an effecton the content

Reply:
This issue seems to be happening in every storybook component that used markdown

Reply:
done